docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1178,10 +1178,27 @@ engineBuilder.addValueResolver(ValueResolver.builder()
 
 Manual registration is sometimes handy but it's also possible to register a template locator using `EngineBuilder.addLocator()`.
 This locator is used whenever the `Engine.getTemplate()` method is called and the engine has no template for a given id stored in the cache.
-The locator is responsible to use the correct character encoding when reading the contents of a template.
+The locator is responsible to use the correct character encoding when reading the contents of a template. The locator must be annotated with
+`io.quarkus.qute.Locate` for each template it locates to disable template validation at build time.
```

Comment:
I think that it's a good practice to start a new sentence on a new line... or at least it's more readable and asciidoc does not add a new line formatting anyway ;-)